tests/BagsExtTests.tla

```diff
@@ -0,0 +1,31 @@
+---- CONFIG BagsExtTests ----
+INIT Init
+NEXT Next
+=====
+
+---------------------------- MODULE BagsExtTests ----------------------------
```

Comment:
BagsExtTests has to be added to https://github.com/tlaplus/CommunityModules/blob/master/tests/AllTests.tla to be included in the "test suite".

Reply:
Thanks, done.